apps/test/unit/templates/lessonOverview/StyledCodeBlockTest.jsx

```diff
@@ -58,4 +58,21 @@ describe('StyledCodeBlock', () => {
       '[`playSound`](/docs/applab/playSound)'
     );
   });
+
+  it('embeds block if blockName is provided', () => {
```

Comment:
This is excellent documentation for me who has no context about this stuff. What happens if blockName is not provided?

Reply:
Oh, based on the other tests, I'm guessing it "builds a full visual code block markdown expression"?

Reply:
Yeah, if no blockName is provided, we build some custom markdown. Frankly, that markdown doesn't work for spritelab docs but is used pretty extensively in CSD (https://studio.code.org/s/csd3-2021/lessons/3 as an example)